src/main/java/uk/gov/justice/hmpps/prison/repository/AgencyRepository.java

```diff
@@ -164,6 +172,23 @@ public List<Location> getAgencyLocationsBooked(final String agencyId, final Loca
         return locations.stream().map(LocationDto::toLocation).collect(toList());
     }
 
+    public Optional<AgyPrisonerPayProfile> getAgyPrisonerPayProfile(final String agencyId) {
```

Comment:
since this looks like a very simple table, would have been far better to map using JPA rather than increasing the amount of this type of code that we're trying to gradually remove

Reply:
Now done.